src/handle_action.cpp

```diff
@@ -2848,11 +2849,12 @@ bool game::handle_action()
         ctxt = get_player_input( action );
     }
 
-#if defined(TILES)
     // Remove asynchronous animations if any action taken before the input timeout
     // Otherwise repeated input can cause animations to accumulate as the timeout is never reached
+#if defined(TILES)
     tilecontext->void_async_anim();
 #endif
+    g->void_async_anim_curses();
```

Comment:
And here.

Reply:
Looks like they do.  Added your changes.